dev/doc/MERGING.md

```diff
@@ -76,6 +76,16 @@ documentation and test cases. If not, they should leave a comment on the PR and
 put the approriate label. Otherwise, they are expected to merge the PR using the
 [merge script](/dev/tools/merge-pr.sh).
 
+When CI has a few failures which look spurious, restarting the corresponding
+jobs is a good way of ensuring this was indeed the case.
+To restart a job on Travis, you should connect using your GitHub account;
+being part of the Coq organization on GitHub should give you the permission
+to restart a job on Travis.
```

Comment:
>to restart a job on Travis

This phrase is repeated at the beginning and end of the sentence, it sounds like an echo.

Reply:
Indeed, thanks.